compiler/rustc_query_system/src/query/plumbing.rs

```diff
@@ -271,6 +267,7 @@ where
                 QueryResult::Poisoned => panic!(),
             }
         };
+        cache.complete(key, result, dep_node_index);
```

Comment:
Why isn't this required any more?

Reply:
Do you mean the extra block that I removed? Not sure when it became redundant.